packages/core/src/interpreter.ts

```diff
@@ -302,15 +310,10 @@ export class Interpreter<
         this._complete();
       }
     } catch (err) {
-      // TODO: properly handle errors
-      if (this.observers.size > 0) {
-        this.observers.forEach((observer) => {
-          observer.error?.(err);
-        });
-        this.stop();
```

Comment:
`.stop()` was removed from here. TODO: reevaluate its importance and role here. If anything this should likely call `._stop` to avoid the "A non-root actor cannot be stopped directly." error - unless this throw can only happen in a root actor (it's unlikely that this is actually the case here).

Reply:
note for me: this should stop the actor by using appropriate error procedures - `stop` schedules a stop signal and that shouldn't happen